docs/pages/docs/guides/singletons.mdx

```diff
@@ -0,0 +1,69 @@
+import { Markdown } from '../../../components/Markdown';
+import { Well } from '../../../components/primitives/Well';
+import { RelatedContent } from '../../../components/RelatedContent';
+
+# Singletons
+
+In Keystone, a Singleton is a list type that contains only one entry.
+
+## Configuration
+
+A little about the configuration differences
+
```

Comment:
A thing to call out somewhere, singletons can have relationships *to* things, but can't have relationships *from* something *to* a singleton - this also means that all relationships including a singleton are one-sided only.